geomesa-tools/conf/sfts/adsbx/reference.conf

```diff
@@ -0,0 +1,248 @@
+        { name = "Call"               , json-type = "string"     , path = "$.Call"         , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Call'), null))" }
+        { name = "CallSus"            , json-type = "bool"       , path = "$.CallSus"      , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.CallSus'), null))" }
+        { name = "HasPic"             , json-type = "bool"       , path = "$.HasPic"       , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.HasPic'), null))" }
+        { name = "FlightsCount"       , json-type = "integer"    , path = "$.FlightsCount" , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.FlightsCount'), null))" }
+        { name = "Interested"         , json-type = "bool"       , path = "$.Interested"   , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Interested'), null))" }
+        { name = "Help"               , json-type = "bool"       , path = "$.Help"         , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Help'), null))" }
+        { name = "TrtRaw"             , json-type = "integer"    , path = "$.Trt"          , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Trt'), null))" }
+        { name = "Trt"                                                                     , transform = "try(cacheLookup('trt_codes', $TrtRaw::string, 'code'), null)" }
+        { name = "TTRaw"              , json-type = "string"     , path = "$.TT"           , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.TT'), null))" }
+        { name = "TT"                                                                      , transform = "try(cacheLookup('tt_codes', $0::string, 'code'), null)" }
+        { name = "Talt"               , json-type = "integer"    , path = "$.Talt"         , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Talt'), null))" }
+        { name = "Ttrk"               , json-type = "integer"    , path = "$.Ttrk"         , transform = "withDefault($0, try(cacheLookup('adsbx_static', $Icao::string, 'adsbx.Ttrk'), null))" }
+        { name = "Sat"                , json-type = "bool"       , path = "$.Sat"          }
+        { name = "PosStale"           , json-type = "bool"       , path = "$.PosStale"     }
+        { name = "dtg"                , transform = "withDefault($PosTime, now())"         }
+        { name = "geom"               , transform = "point($Long::double, $Lat::double)"   }
+      ]
+    }
+
+    adsbx = ${geomesa.converters.adsbx-Base} {
+      id-field = "md5(stringToBytes(concatenate($Icao::string, $dtg::string, $Lat::string, $Long::string)))"
+      //feature-path = "$.acList[*]"
```

Comment:
this one and adsbx-single are the same, right? maybe we should make this the default, then add one for adsbx-multi that has the feature path uncommented.